### PR TITLE
Update build tools for mjml-react and release the first version

### PR DESCRIPTION
**Pull request by harry-wang-faire**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,77 +1,62 @@
...
-          "SwitchCase": 1,
-          "ignoredNodes": [
-            "JSXElement",
-            "JSXElement *"
-          ]
-        }
-      ]
-    }
+  "dependencies": {
+    "color": "^3.1.3"
```
Comment (IanEdington) [inline package.json:60]: Is this a true dependency? As in it's needed by all instances of mjml-react in order to work properly in someone elses build?

File: package.json [hunk 2]
```diff
@@ -1,77 +1,62 @@
...
+    "@babel/preset-env": "^7.19.0",
+    "@babel/preset-react": "^7.18.6",
+    "chai": "^4.3.6",
+    "jest": "^29.0.3",
+    "mjml": "^4.7.0",
+    "mjml2json": "^1.0.1",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
     "react-reconciler": "^0.26.1"
```
Comment (IanEdington) [inline package.json:57]: Is "react-reconciler" needed as a peer dependency? It looks like it used to be a dependency

File: .npmrc [hunk 3]
```diff
@@ -1 +1,4 @@
-package-lock=false
\ No newline at end of file
+//registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
+registry=https://registry.npmjs.org/
+always-auth=true
```
Comment (IanEdington) [inline .npmrc:3]: I think this will need to be removed when other people start working on the project, but that can be tackled at a different time.

**Review by IanEdington — APPROVED** [review]
Looks great!

File: package.json (hunk 1 above)
Comment (harry-wang-faire) [reply package.json]: Yes it is a dependency. There is a script that relies on `color` import. We can clean up this in the future but let's keep it as close to the existing release as possible

File: package.json (hunk 2 above)
Comment (harry-wang-faire) [reply package.json]: I just put back to wherever it should belong in the previous version of `package.json`

File: .npmrc (hunk 3 above)
Comment (harry-wang-faire) [reply .npmrc:3]: Sounds good, let's revisit this in the future

File: package.json (hunk 1 above)
Comment (IanEdington) [reply package.json]: If it's a script that isn't exported in the npm `bin` we can probably make it dev only

Comment (harry-wang-faire) [reply package.json]: It is exported in `index.js` 
`export { render, renderToMjml, renderToJSON, renderToJSON2 };`
`renderToJSON` is the consumer

**Pull request merged**